Formula/dbus.rb

```diff
@@ -30,16 +36,16 @@ def install
     # Fix the TMPDIR to one D-Bus doesn't reject due to odd symbols
     ENV["TMPDIR"] = "/tmp"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--localstatedir=#{var}",
-                          "--sysconfdir=#{etc}",
-                          "--disable-xml-docs",
-                          "--disable-doxygen-docs",
-                          "--enable-launchd",
-                          "--with-launchd-agent-dir=#{prefix}",
-                          "--without-x",
-                          "--disable-tests"
+    system "./autogen.sh", "--disable-dependency-tracking",
```

Comment:
@aw1621107 configure is generated by autogen. Sometimes a "release tarball" will already have it pre-generated for you. It's a distinction between building from the raw source code repository and building from a packaged release of the sources.
```
Josephs-MacBook-Pro:~ joe$ tar -tf <(curl -sL https://dbus.freedesktop.org/releases/dbus/dbus-1.10.8.tar.gz)|grep configure
dbus-1.10.8/configure.ac
dbus-1.10.8/configure
Josephs-MacBook-Pro:~ joe$ tar -tf <(curl -sL https://dbus.freedesktop.org/releases/dbus/dbus-1.11.4.tar.gz)|grep configure
dbus-1.11.4/configure.ac
dbus-1.11.4/configure
```
Whereas in the git repository:
```
$ ls dbus|grep configure
configure.ac
```
After running autogen, you will have configure just like the tarball.